<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoHuntConfiguratorClass</class>
 <widget class="QDialog" name="AutoHuntConfiguratorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoHuntConfigurator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="0">
    <widget class="QGroupBox" name="attackGroup">
     <property name="title">
      <string>Ataki</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="1">
         <widget class="QLabel" name="constLabel">
          <property name="text">
           <string>min. l. potworów</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="actvieAtackCheckBox_1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="attackMinMonsers_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="constLabel2">
          <property name="text">
           <string>atak</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="attackMinMonsers_1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="actvieAtackCheckBox_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="actvieAtackCheckBox_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QComboBox" name="atakNameBox_4">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="attackMinMonsers_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="actvieAtackCheckBox_4">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QComboBox" name="atakNameBox_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="atakNameBox_1">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QComboBox" name="atakNameBox_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="attackMinMonsers_4">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9</number>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QComboBox" name="atakKeyBox_4">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QComboBox" name="atakKeyBox_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QComboBox" name="atakKeyBox_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QComboBox" name="atakKeyBox_1">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="constLabel3">
          <property name="text">
           <string>przycisk</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="RouteGroip">
     <property name="title">
      <string>Trasa</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="0">
       <widget class="QPushButton" name="selectRouteButton">
        <property name="text">
         <string>Wybierz trasę</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="editRouteButton">
        <property name="text">
         <string>Edytor tras</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="routeConstLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Trasa: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="routeLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>-------</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="4" rowspan="10">
    <spacer name="horizontalSpacer">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="monstersAmountGroup">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="minMonStopConstLabel">
        <property name="text">
         <string>Min. liczba potworów do zatrzymania się</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="minMonStopSpiner">
        <property name="maximum">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="minMonContinueConstLabel">
        <property name="text">
         <string>Min. liczba potworów do kontynuwania trasy</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="minMonContinueSpiner">
        <property name="maximum">
         <number>7</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>anuluj</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="finishButton">
       <property name="text">
        <string>zapisz i wyjdź</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>finishButton</sender>
   <signal>clicked()</signal>
   <receiver>AutoHuntConfiguratorClass</receiver>
   <slot>saveButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>AutoHuntConfiguratorClass</receiver>
   <slot>cancelButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actvieAtackCheckBox_1</sender>
   <signal>stateChanged(int)</signal>
   <receiver>AutoHuntConfiguratorClass</receiver>
   <slot>checkBoxChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actvieAtackCheckBox_2</sender>
   <signal>stateChanged(int)</signal>
   <receiver>AutoHuntConfiguratorClass</receiver>
   <slot>checkBoxChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actvieAtackCheckBox_3</sender>
   <signal>stateChanged(int)</signal>
   <receiver>AutoHuntConfiguratorClass</receiver>
   <slot>checkBoxChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actvieAtackCheckBox_4</sender>
   <signal>stateChanged(int)</signal>
   <receiver>AutoHuntConfiguratorClass</receiver>
   <slot>checkBoxChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveButtonClicked()</slot>
  <slot>cancelButtonClicked()</slot>
  <slot>checkBoxChanged()</slot>
 </slots>
</ui>
